_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Reporting CLI
+nav_order: 25
+---
+
+# Reporting CLI
+
+You can download OpenSearch Dashboards reports directly through the reporting CLI. You don't need to use the OpenSearch Dashboards or the Reporting plugin to download reports. You choose the file format that you want and specify either PNG, PDF or CSV file format with the `--formats` argument.
```

Comment:
For CSV reports, the reporting plugin should be installed. Not sure don't need to use reporting plugin implies, user can uninstall the plugin.